doc/languages-frameworks/python.section.md

```diff
@@ -1445,24 +1447,26 @@ referencing it using `sphinxHook` from top-level.
 
 ### Develop local package {#develop-local-package}
 
-As a Python developer you're likely aware of [development mode](http://setuptools.readthedocs.io/en/latest/setuptools.html#development-mode)
-(`python setup.py develop`); instead of installing the package this command
-creates a special link to the project code. That way, you can run updated code
-without having to reinstall after each and every change you make. Development
-mode is also available. Let's see how you can use it.
+As a Python developer you're likely aware of [development mode]
+(https://setuptools.pypa.io/en/latest/userguide/development_mode.html)
```

Comment:
This line break within the link does not render as an anchor in neither github nor with `pandoc --from commonmark`

Reply:
There are multiple instances of this

Reply:
Editor got over-eager with word-wrapping...